ONSdigital/eq-author-app
# Ear 2282 repeating title

Update section title to support piping in repeating sections

To test -
- Add a list with multiple fields
- Link repeating section to list
- pipe multiple fields into the section title
- Delete/amend/unlink list and check validation

Repeat for supplementary data
